java/libphonenumber/pom.xml

```diff
@@ -42,11 +42,32 @@
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
         <version>3.0.2</version>
-        <configuration>
-          <archive>
-            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-          </archive>
-        </configuration>
+        <executions>
+          <execution>
+            <id>default-jar</id>
```

Comment:
I followed the [maven-jar-plugin example](https://maven.apache.org/plugins/maven-jar-plugin/examples/attached-jar.html); I don't believe these are necessary to add for the default jar configuration, but are required for adding an attached classified jar.